hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Reconciler.java

```diff
@@ -117,4 +124,42 @@ private void explicitlyReconcileTasks(SchedulerDriver driver) {
 
     driver.reconcileTasks(tasks);
   }
+
+  private class ReconcileThread extends Thread {
+    private static final int MILLIS_FROM_SECONDS = 1000;
```

Comment:
Why should it be put in the constants file?  Not all constants should go there.  This constant is used only once.  There is value in keeping it near its place of usage.  It exists only to clarify intent.  It does not have relevance to the HDFS framework apart from that and does not belong in HdfsConstants.  You are thinking of an old constant which was incorrectly named SECONDS_FROM_MILLIS which was in the HdfsScheduler.java.  It's not there anymore, so there are no duplicate constants.

This is not my first day writing software.  I am aware that copy/pasting constants around is a bad idea.